Tasks/DotNetCoreInstallerV0/typings/globals/mocha/index.d.ts

```diff
@@ -1,202 +0,0 @@
-// Generated by typings
```

Comment:
typings is the old thing, I switched to `@types` as part of this change (see package.json below). I was hitting a typescript compile error after referencing the new vsts-task-tool-lib version.

Reply:
and `@types` is the way forward anyway... so this seemed like a good time to switch over